namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-complexity-of-need-staging/resources/ecr.tf

```diff
@@ -0,0 +1,24 @@
+module "ecr-repo-complexity-of-need" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.1"
```

Comment:
The ECR module is on v4.3 now. Please use the latest version.

Reply:
updated as requested